frontend/src/router.ts

```diff
@@ -7,6 +7,10 @@ const routes: Array<RouteRecordRaw> = [
     path: "/",
     component: () => import("./views/OptionsView.vue")
   },
+  {
+    path: "/about",
+    component: () => import("./pages/About.vue")
```

Comment:
Please move _/pages/About.vue_ into _view/pages/AboutView.vue_ for consistency (all the router views are in views)

Reply:
done.